src/fread.c

```diff
@@ -1495,7 +1495,7 @@ int freadMain(freadMainArgs _args) {
           const char *prevLineStart=ch, *lineStart=ch;
           int lastncol = countfields(&ch);
           if (lastncol<0) continue;  //   invalid file with this sep and quote rule, skip
-          ASSERT(lastncol>0, "Internal error: first non-empty row should always be at least one field; %c %d", sep, quoteRule); // # nocov
+          ASSERT(lastncol>0, _("Internal error: first non-empty row should always be at least one field; %c %d"), sep, quoteRule); // # nocov
```

Comment:
OK, so I figured out what's going wrong here. And now I remember that I had found something wrong here earlier but gave up before understanding what's the issue...

`ASSERT` is doing `char` array literal concatenation:

```
"Internal error in line %d of fread.c, please report on data.table GitHub:  " msg
```

However, this breaks when we do the translation -- note (from `po.h`) that `_(String)` is expanding to `dgettext("data.table", String)`, so if we wrap the `ASSERT` fixed literal with whatever the message is, after macro expansion we'll have:

```
dgettext("data.table", "Internal error in line %d of fread.c, please report on data.table GitHub:  ") dgettext("data.table", msg)
```

The `f() g()` is a syntax error.

I suppose we _could_ complicate things by doing some array copying, etc., to handle this more properly, BUT seeing as how these are all _internal_ errors in the first place (meaning users should never see them, in principle), I don't think it's worth the effort.

So I'll roll back the use of translation with `ASSERT`. Sorry about that!